### ISSUE TITLE
Functions that take more than one parameter are not called correctly

### ISSUE THREAD

**Issue by 109C** [bobbybee/scratch-llvm#2]
The first one is correct, but the rest of them have prefixes 

![function call with two](https://cloud.githubusercontent.com/assets/9680886/8139539/8b321734-1122-11e5-8a35-e3d37b6899a1.png)
![function call with three](https://cloud.githubusercontent.com/assets/9680886/8139540/8beda652-1122-11e5-9388-d8053e232b0a.png)
```
int MultiCall(int p1, int p2, int p3){
  return p1+p2+p3;
}
```

I *think* this has something to do with the regexes, but I haven't been able to fix it.

**Comment by bobbybee** [comment]
Eek. I'll look into it.

**Comment by bobbybee** [comment]
Heh, silly me. Closed by 2f98612ea19c9123cebeae50f37486564d935aad

**Comment by bobbybee** [comment]
@109C You were close, actually :)
